examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -9,152 +9,346 @@ import '../../gallery/demo.dart';
-              Expanded(
-                child: Padding(
-                  padding: const EdgeInsets.fromLTRB(16.0, 16.0, 16.0, 0.0),
-                  child: DefaultTextStyle(
-                    softWrap: false,
-                    overflow: TextOverflow.ellipsis,
-                    style: descriptionStyle,
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: <Widget>[
-                        // three line description
-                        Padding(
-                          padding: const EdgeInsets.only(bottom: 8.0),
-                          child: Text(
-                            destination.description[0],
-                            style: descriptionStyle.copyWith(color: Colors.black54),
-                          ),
-                        ),
-                        Text(destination.description[1]),
-                        Text(destination.description[2]),
-                      ],
-                    ),
-                  ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class SectionTitle extends StatelessWidget {
+  const SectionTitle({
+    Key key,
+    this.title
+  }) : super(key: key);
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(4.0, 4.0, 4.0, 12.0),
+      child: Align(
+        alignment: Alignment.centerLeft,
+        child: Text(title, style: Theme.of(context).textTheme.subhead),
+      ),
+    );
+  }
+}
+
+class TravelDestinationContent extends StatelessWidget {
+  const TravelDestinationContent({ Key key, @required this.destination })
+    : assert(destination != null),
+      super(key: key);
+
+  final TravelDestination destination;
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final TextStyle titleStyle = theme.textTheme.headline.copyWith(color: Colors.white);
+    final TextStyle descriptionStyle = theme.textTheme.subhead;
+
+    // In order to have the ink splash appear above the child image, you must
+    // use Ink.image. This allows the image to become part of the Material and
+    // display ink effects above it. Using a standard Image will obscure the ink
+    // splash.
+    final Widget image = destination.type == CardDemoType.standard
+      ? Image.asset(
+          destination.assetName,
+          package: destination.assetPackage,
+          fit: BoxFit.cover,
+        )
+      : Ink.image(
+          image: AssetImage(destination.assetName, package: destination.assetPackage),
+          fit: BoxFit.cover,
+          child: Container(),
+        );
+
+    final List<Widget> children = <Widget>[
+      // Photo and title.
+      SizedBox(
+        height: 184.0,
+        child: Stack(
+          children: <Widget>[
+            Positioned.fill(
+              child: image,
```

Comment:
Can we make this widget all on one line/

Reply:
I inlined the image widget after removing the ternary so this isn't applicable anymore